src/applications/disability-benefits/686/config/form.js

```diff
@@ -13,7 +13,6 @@ import fullNameUI from 'platform/forms/definitions/fullName';
 import dataUtils from 'platform/utilities/data/index';
 import environment from 'platform/utilities/environment';
 import { externalServices } from 'platform/monitoring/DowntimeNotification';
-import { VA_FORM_IDS } from 'platform/forms/constants';
```

Comment:
This is the old 686 form, which is not currently in use nor maintained. I moved the use of the formId constant from here to the new 686 form.